examples/access-control-migration/package.json

```diff
@@ -29,7 +29,9 @@
     "test:dev": "lb-mocha --allow-console-logs dist/__tests__/**/*.js",
     "verify": "npm pack && tar xf loopback-access-control-migration*.tgz && tree package && npm run clean",
     "migrate": "node ./dist/migrate",
+    "premigrate": "npm run build ",
     "openapi-spec": "node ./dist/openapi-spec",
+    "preopenapi-spec": "npm run build",
```

Comment:
Nitpick: 

- move `premigrate` to come before `migrate`
- move `preopenapi-spec` to come before `openapi-spec`

Reply:
Done.